testproject/Assets/Scripts/PlayerMovement.cs

```diff
@@ -38,12 +38,24 @@ public override void OnNetworkSpawn()
         Players[OwnerClientId] = this; // todo should really have a NetworkStop for unregistering this...
     }
 
+    [ServerRpc(RequireOwnership = true)]
```

Comment:
why do we need a server RPC? shouldn't this still work with the new client network transform?